src/test/java/com/ing/data/cassandra/jdbc/metadata/CompleteMetadataTest.java

```diff
@@ -0,0 +1,110 @@
+
+    @Container
+    private final CassandraContainer<?> dbContainer =
+            new CassandraContainer<>(imageName.withTag("5.0")).withInitScript("initEmbeddedDse.cql");
+
+    private DatabaseConnectionSource dbConnectionSource;
+
+    /****
+        * Sets up a data source to connect to the Cassandra database.
+        *
+    ****/
+    @BeforeEach
+    public void createDatabase() {
+
+        if (!dbContainer.isRunning()) {
+            fail("Testcontainer for database is not available");
+        }
+
+        final InetSocketAddress contactPoint = dbContainer.getContactPoint();
+        final String host = contactPoint.getHostName();
+        final int port = contactPoint.getPort();
+        final String keyspace = "test_keyspace_vect";
+        final String localDatacenter = dbContainer.getLocalDatacenter();
+        final String connectionUrl = String.format("jdbc:cassandra://%s:%d/%s?localdatacenter=%s", host,
+                port, keyspace, localDatacenter);
+        System.out.printf("url=%s%n", connectionUrl);
+        createDataSource(connectionUrl);
+    }
+
+    /****
+        * Uses SchemaCrawler to obtain database metadata in "maximum" mode. 
+        * Compares actual results to an expected file with results. The test will
+        * produce a new expected results file if the actual and expected results
+        * do not match.
+		*
+		* @throws Exception on an error when running the test
+    ****/
+    @Test
+    public void givenDatabase_whenCompleteMetadataExtracted_shouldMatchExpectedOutput()
+            throws Exception {
+
+        final LimitOptionsBuilder limitOptionsBuilder =
+                LimitOptionsBuilder.builder().includeSchemas(Pattern.compile("test_keyspace_vect"));
+        final SchemaInfoLevelBuilder schemaInfoLevelBuilder =
+                SchemaInfoLevelBuilder.builder().withInfoLevel(InfoLevel.maximum);
+        final LoadOptionsBuilder loadOptionsBuilder =
+                LoadOptionsBuilder.builder().withSchemaInfoLevelBuilder(schemaInfoLevelBuilder);
+        final SchemaCrawlerOptions schemaCrawlerOptions = SchemaCrawlerOptionsBuilder
+                .newSchemaCrawlerOptions().withLimitOptions(limitOptionsBuilder.toOptions())
+                .withLoadOptions(loadOptionsBuilder.toOptions());
+        final SchemaTextOptionsBuilder textOptionsBuilder = SchemaTextOptionsBuilder.builder();
+        textOptionsBuilder.showDatabaseInfo().showJdbcDriverInfo();
+        final SchemaTextOptions textOptions = textOptionsBuilder.toOptions();
+
+        final SchemaCrawlerExecutable executable = new SchemaCrawlerExecutable("details");
+        executable.setSchemaCrawlerOptions(schemaCrawlerOptions);
+        executable.setAdditionalConfiguration(SchemaTextOptionsBuilder.builder(textOptions).toConfig());
+
+        final String expectedResource = "expected_metadata_output.txt";
+        assertThat(outputOf(executableExecution(getDataSource(), executable)),
```

Comment:
For now, the test failed here when the CI pipeline runs.
```
Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 9.404 s <<< FAILURE! - in com.ing.data.cassandra.jdbc.metadata.CompleteMetadataTest
Error:  givenDatabase_whenCompleteMetadataExtracted_shouldMatchExpectedOutput  Time elapsed: 9.402 s  <<< ERROR!
schemacrawler.schemacrawler.exceptions.ExecutionRuntimeException: java.lang.AssertionError
	at com.ing.data.cassandra.jdbc.metadata.CompleteMetadataTest.givenDatabase_whenCompleteMetadataExtracted_shouldMatchExpectedOutput(CompleteMetadataTest.java:97)
Caused by: us.fatehi.utility.scheduler.RunnerException: java.lang.AssertionError
	at com.ing.data.cassandra.jdbc.metadata.CompleteMetadataTest.givenDatabase_whenCompleteMetadataExtracted_shouldMatchExpectedOutput(CompleteMetadataTest.java:97)
Caused by: java.lang.AssertionError
```
(See here: https://github.com/ing-bank/cassandra-jdbc-wrapper/actions/runs/6867960716/job/18677416806#step:4:2854)

Could you try to update your branch to include the fix #39 and see if this fixes the test?
I didn't go deeply in the SchemaCrawler code to understand how the `hasSameContentAs` method works, but does it **strictly** compare the test output with the expected one? (because the `generated on` value will be systematically different for example).